internal/integration/fixtures/extra-volume-mounts.yaml

```
@@ -0,0 +1,30 @@
+steps:
+  - label: ":wave:"
+    agents:
+      queue: {{.queue}}
+    env:
+      BUILDKITE_GIT_CLONE_MIRROR_FLAGS: --mirror
+      BUILDKITE_GIT_MIRRORS_PATH: /tmp/buildkite-git-mirrors
+    plugins:
+      - kubernetes:
+          podSpec:
+            containers:
+              - image: alpine:latest
+                command: [ash]
+                args:
+                  -  -c
+                  - "echo 'volume mounted' > /tmp/buildkite-git-mirrors/foo.txt"
+              - image: alpine:latest
+                command: [ash]
+                args:
+                  - -c
+                  - "sleep 2; cat /tmp/buildkite-git-mirrors/foo.txt"
```

```suggestion
                  - '"sleep 2; cat /tmp/buildkite-git-mirrors/foo.txt"'
```
Turns out, when the YAML is parsed, the double quotes are dropped. But they are needed so that `ash -c` receives one argument. I think escaping them will fix this.